athena-jdbc/src/main/java/com/amazonaws/connectors/athena/jdbc/saphana/SAPHANACompositeHandler.java

```diff
@@ -0,0 +1,39 @@
+/*-
+ * #%L
+ * athena-jdbc
+ * %%
+ * Copyright (C) 2019 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.connectors.athena.jdbc.saphana;
+
+import com.amazonaws.athena.connector.lambda.handlers.CompositeHandler;
+import com.amazonaws.athena.connector.lambda.handlers.MetadataHandler;
+import com.amazonaws.athena.connector.lambda.handlers.RecordHandler;
+
+/**
+ * Boilerplate composite handler that allows us to use a single Lambda function for both
+ * Metadata and Data. In this case we just compose {@link SAPHANAMetadataHandler} and {@link SAPHANARecordHandler}.
+ *
+ * Recommend using {@link com.amazonaws.connectors.athena.jdbc.MultiplexingJdbcCompositeHandler} instead.
+ */
+public class SAPHANACompositeHandler
```

Comment:
This doesnt follow our usual naming convention. Can you rename to SapCompositeHandler?

Reply:
SAP had multiple database products - SAP HANA, SAP AES, SAP MaxDB, SAP IQ and others. in future we may add support for other SAP databases as well. So i think this is required.

Reply:
Thats fine, choose a name that uses CammelCase

Reply:
Fixed, please check